amp/TEMPLATE/reamp/modules/activity/preview/api/ActivityApi.jsx

```diff
@@ -0,0 +1,48 @@
+import {fetchJson, postJson} from 'amp/tools/index';
+import {ACTIVITY_API, FIELDS_DEFINITION_API, POSSIBLE_VALUES_API, FM_API} from '../common/Constants.jsx'
+
+
+export default class ActivityApi {
+    static getActivity(activityId) {
+        return new Promise((resolve, reject) => {
+            fetchJson(ACTIVITY_API + activityId).then((activity) => {
```

Comment:
my ES6 is a bit rusty, could you confirm if in these cases is not necessary to put a return before beforeJson?

Reply:
I think we do, let me check. Also we could enable eslint but only for the preview module